src/Home.js

```diff
@@ -52,12 +66,47 @@ export default function Home() {
 
   return (
     <View style={styles.container}>
+      <Modal
```

Comment:
We may want to discuss moving some of these components in to smaller sub-components, what do you think?

Reply:
sure caleb, we can discuss this in today's meeting.